crates/server/src/admin/handlers/alias/list_aliases.rs

```diff
@@ -1,37 +1,39 @@
+use std::collections::HashMap;
 use std::sync::Arc;
 
 use axum::extract::Path;
 use axum::response::IntoResponse;
 use axum::Extension;
-use calimero_server_primitives::admin::{AliasRecord, ListAliasesResponse};
+use calimero_primitives::alias::Alias;
+use calimero_server_primitives::admin::ListAliasesResponse;
 use calimero_store::key::{Aliasable, StoreScopeCompat};
-use serde::{Deserialize, Serialize};
+use reqwest::StatusCode;
+use serde::Serialize;
 
-use crate::admin::service::{parse_api_error, ApiResponse};
+use crate::admin::service::ApiResponse;
 use crate::AdminState;
 
 pub async fn handler<T>(
     Extension(state): Extension<Arc<AdminState>>,
     maybe_scope: Option<Path<T::Scope>>,
 ) -> impl IntoResponse
 where
-    T: Aliasable + Serialize + Clone + From<[u8; 32]>,
-    T::Scope: Copy + PartialEq + StoreScopeCompat,
+    T: Aliasable<Scope: StoreScopeCompat + Copy + PartialEq> + Serialize + From<[u8; 32]> + Eq,
```

Comment:
when associated type bounds are more than 1 and the bounded type also has trait constraints, breaking them up is advisable